test/defineInput.spec.ts

```diff
@@ -0,0 +1,28 @@
+import {defineInput} from '../src/utils/defineInput'
```

Comment:
This file needs cleanup, perhaps add the test dir to be included in `npm run lint` too